src/components/common/EthHashInfo/SrcEthHashInfo/index.tsx

```diff
@@ -62,7 +62,7 @@ const SrcEthHashInfo = ({
   )
 
   return (
-    <div className={css.container}>
+    <div className={css.container} style={{ width: '100%' }}>
```

Comment:
Could you move thos style it to styles.module.css to the `.container` class?

Reply:
Done! Had not noticed it.